recipe/meta.yaml

```
@@ -18,14 +18,13 @@ build:
     - image_to_iq = xpdtools.cli.process_tiff:run_main
 
 requirements:
-  host:
+  build:
```

Why was it changed to `build`? See https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#host for details. I am going to revert it back, and merge after successful build.
```suggestion
  host:
```